docs/resources/integration_log.md

```diff
@@ -289,7 +289,7 @@ The following arguments are supported:
 * `access_key_id`     - (Optional/Sensitive) AWS access key identifier.
 * `secret_access_key` - (Optional/Sensitive) AWS secret access key.
 * `api_key`           - (Optional/Sensitive) The API key.
-* `tags`              - (Optional) Tag the integration, e.g. env=prod,region=europe.
+* `tags`              - (Optional) Tags. e.g. `env=prod,region=europe,version=v1.0`. The "v" prefix is required for version.
```

Comment:
```suggestion
* `tags`              - (Optional) Tags. e.g. `env=prod,region=europe`. 

 ***Note: If tags are used with Datadog. The value part (prod, europe, ...) must start with a letter, read more about tags format in the [Datadog documentation](https://docs.datadoghq.com/getting_started/tagging/#define-tags)***
```
Would rather say that it's not a requirement for version per se. Rather when using Datadog and for each value part in the key-value pair. Similar for metrics integrations.